### ISSUE TITLE
Delete item button doesn't work

### ISSUE THREAD

**Issue by muhep06** [DSpace/dspace-angular#1481]
I am using Dspace version 7.1. When I try to permanently delete the item, nothing happens. Delete button not working properly. Also, sometimes when I click the permanently delete button on the edit item page, it throws a "null" request.
``https://[domain]/server/api/core/entitytypes/label/null``

![image](https://user-images.githubusercontent.com/6313999/147739376-dc6b7c30-6989-4b0d-801a-53fd1e6969ef.png)

**Comment by tdonohue** [comment]
@muhep06 : I cannot reproduce this on our Demo site (https://demo7.dspace.org/), or locally in the latest code on `main` (which does include code updates since the 7.1 release).  So, it appears to me that we've fixed this bug already, and the fix will be available in the upcoming 7.2 release (coming in Feb)

**Comment by firoball** [comment]
I can confirm this issue happening with DSpace 7.1.1 when working as administrator (creation and deletion).

**Comment by muhep06** [comment]
I fix this issue insert entity models into database with using this command.
```bash
[dspace]/bin/dspace initialize-entities -f [dspace]/config/entities/relationship-types.xml
```
https://wiki.lyrasis.org/display/DSDOC7x/Configurable+Entities#ConfigurableEntities-DefaultEntityModels

Then i select entity type for new collection.
![image](https://user-images.githubusercontent.com/6313999/149628695-268ba9a4-e80d-411a-8c6b-e89d199e45fb.png)

After all, i try to permanently delete again it works.

The entity type field must be required if permanent deletion will not work if I don't select an entity type for the collection.

**Comment by firoball** [comment]
Thanks for the hint. With items (not collections) I have not been successful so far, though.

**Comment by tdonohue** [comment]
@firoball and @muhep06 : thanks for the additional discussion here.  It appears I missed that this bug was related to the existence of an empty `dspace.entity.type` field.   That said, I'll leave this ticket closed, as it's now been duplicated into #1501.

However, you'll both be glad to hear **we have a PR to fix this issue in #1503.** 

So, this bug will be fixed on `main` shortly & will be released in 7.2

**Comment by firoball** [comment]
I have already added this new PR into our local setup and can confirm this fixes the issue for me without leftovers.
